src/styles/App.scss

```diff
@@ -135,3 +135,17 @@ a {
     margin: 12px;
     text-decoration: underline;
 }
+.home {
+    &__page {
+        h1 {
+            font-size: 20px;
+            color: yellow;
+            // margin-left: 20px;
+        }
+
+    }
+}
+.slider {
+    // padding: 1em;
+    padding-bottom: 3em;
+}
```

Comment:
good practice to add new line in the end of the file. not really necessary in most cases but its done to prevent issues if some concatenation is applied